File: fibonacci series.cpp
#include<iostream>
using namespace std;
main()
{
    int a,b,c,n;
    a=0;
    b=1;
    cout<<"Enter the value of n:";
    cin>>n;
    cout<<"Fibonacci series upto n is:"<<endl;
    cout<<a<<endl;
    cout<<b<<endl;
    c=a+b;
    while(c<=n)
    {
        cout<<c<<endl;
        a=b;
        b=c;
        c=a+b;
    }
}

File: fibonacci series.py
n=int(input("Enter the number upto which you want to find the series:"))
a=0
b=1
i=0
print(a)
print(b)
while(i<n-2):
    c=a+b
    if(c<=n):
        print(c)
        a=b
        b=c
    else:
        print("End of loop!")
        break
    i+=1

